docker-compose/docker-compose.yaml

```diff
@@ -510,6 +512,30 @@ services:
       - sourcegraph
     restart: always
 
+  # Description: TimescaleDB time-series database for code insights data.
+  #
+  # Disk: 128GB / persistent SSD
+  # Network: 1Gbps
+  # Ports exposed to other Sourcegraph services: 5432/TCP 9187/TCP
+  # Ports exposed to the public internet: none
+  # 
+  # Note: You should deploy this as a container, do not try to connect it to your external
+  # Postgres deployment (TimescaleDB is a bit special and most hosted Postgres deployments
+  # do not support TimescaleDB, the data here is akin to gitserver's data, where losing it
+  # would be bad but it can be rebuilt given enough time.)
+  codeinsights-db:
+    container_name: codeinsights-db
+    image: "index.docker.io/sourcegraph/codeinsights-db:insiders@sha256:f985af2fef860cc48be40ded864df025b8794b02b86e66cbc6c55bfe3c418831"
+    cpus: 4
+    mem_limit: "2g"
+    environment:
+      - POSTGRES_PASSWORD=password
+    volumes:
+      - "codeinsights-db:/data/"
+    networks:
+      - sourcegraph
+    restart: always
```

Comment:
Is there any reason we didn't add health checks here?

Reply:
no good reason I can think of, probably I just forgot to do it.